TTHAnalysis/python/plotter/susy-1lep/1l_basicCutsFullCutFlow.txt

```diff
@@ -1,21 +1,35 @@
-1 hard lep: nLep == 1 && Lep_pt > 25 && nVeto == 0
+# Select events with exactly one lepton
+1 hard lep: nLep == 1 && Lep_pt > 25
+
+# Veto events with additional leptons
+No veto lep: nVeto == 0
+
+# Lepton needs to be selected
 Selected: Selected == 1
 
-Trigger: !isData || (isData && (HLT_EleHT350 || HLT_EleHT400 || HLT_Ele105 || HLT_MuHT400 || HLT_MuHT350))
+# Fire at least one trigger
+Trigger: !isData || (HLT_EleHT350 || HLT_EleHT400 || HLT_Ele105 || HLT_MuHT400 || HLT_MuHT350)
 
-# use the following line to pick leptons only from the corresponding PD
-XOR: !isData || (isData && ((PD_SingleEle && nEl==1) || (PD_SingleMu && nMu==1)))
+# Prevent double counting from different datasets
+Trigger XOR: !isData || ((PD_SingleEle && (HLT_EleHT350 || HLT_EleHT400 || HLT_Ele105)) || (PD_SingleMu && (HLT_MuHT400 || HLT_MuHT350) && !((HLT_EleHT350 || HLT_EleHT400 || HLT_Ele105))))
```

Comment:
Logic looks good to me. 
However, there are triggers missing (Ele105 became prescaled; Ele115 need to be ORed as well) + we will have to update this again with the MET dataset.